projects/js-packages/components/components/product-offer/index.jsx

```diff
@@ -123,7 +125,8 @@ const ProductOffer = ( {
 					products={ supportedProducts?.length ? supportedProducts : [ slug ] }
 				/>
 				<H3>{ title }</H3>
-				<Text mb={ 3 }>{ description }</Text>
+				{ subTitle && <Title mb={ 3 }>{ subTitle }</Title> }
```

Comment:
For me looks really strange use a component `Title` with `subTitle`, but don't think we have a better approach for that, just adding the discussion for more ideas (if someone has).

Reply:
Same thoughts.
Probably we should consider to changing component names and or properties names.
These changes are based on the design:

<img src="https://user-images.githubusercontent.com/77539/164223862-444a17f7-ab6a-457f-adc8-1b1fd681796f.png" width="400px" />